finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java

```diff
@@ -33,7 +33,11 @@
   // end::annotations[]
   @GET
   @Produces(MediaType.APPLICATION_JSON)
+  // tag::getProperties[]
   public Properties getProperties()
+      // tag::IOException[]
       throws UnknownUrlException, IOException, ProcessingException;
+      // end::IOException[]
 }
-// end::client[]
+// end::getProperties[]
```

Comment:
`getProperties()` hotspot is highlighting too much (the original guide is highlighting it wrong too). 
It should only be highlighting 2 lines. Should only highlight the function:
```
public Properties getProperties()
      throws UnknownUrlException, IOException, ProcessingException;
```

Reply:
> Additional stuff (not in review comments):
> 
>     * [ ]  `microprofile` hotspot should now highlight entire dependency

do you mean from start tag to end tag?
<img width="520" alt="image" src="https://user-images.githubusercontent.com/25497083/70011134-dee71d80-1540-11ea-8aae-ef299195e2e7.png">

Reply:
> `getProperties()` hotspot is highlighting too much (the original guide is highlighting it wrong too).
> It should only be highlighting 2 lines. Should only highlight the function:
> 
> ```
> public Properties getProperties()
>       throws UnknownUrlException, IOException, ProcessingException;
> ```

not quite sure it is highlighting the extra line.

Reply:
> `getProperties()` hotspot is highlighting too much (the original guide is highlighting it wrong too).
> It should only be highlighting 2 lines. Should only highlight the function:
> 
> ```
> public Properties getProperties()
>       throws UnknownUrlException, IOException, ProcessingException;
> ```

Unaddressed, it shouldn't include the end bracket.

Reply:
Addressed.